pyqtgraph/functions.py

```diff
@@ -233,13 +234,19 @@ def mkColor(*args):
     if len(args) == 1:
         if isinstance(args[0], basestring):
             c = args[0]
-            if c[0] == '#':
-                c = c[1:]
             if len(c) == 1:
                 try:
                     return Colors[c]
                 except KeyError:
                     raise ValueError('No color named "%s"' % c)
+            if c[0] == '#':
+                c = c[1:]
+            else:
+                warnings.warn(
+                    "Parsing of hex strings that do not start with '#' is"
+                    "deprecated support will be removed in 0.13",
```

Comment:
--> deprecated *and* support will be removed in 0.13  ??

Reply:
Good suggestion, incorporated.